src/app/profile-setting/image-cropper/image-cropper.component.ts

```diff
@@ -17,7 +17,7 @@ export class ImgCropperComponent {
   profileData: any;
   croppedImageFile: any;
   cropperSettings: CropperSettings;
-  @ViewChild('cropper', undefined)
+  @ViewChild('cropper', { static: false })
```

Comment:
I think static: false is optional in ng9 but static: true is mandatory so you can remove options object here. (try it before committing)

Reply:
I tested this and apparently if I wouldn't use {static: false} we would crash.

Reply:
funny... maybe something is wrong with our code. see docs under description you see default value for static is false: https://angular.io/api/core/ViewChild#description